openedx/core/djangoapps/oauth_dispatch/views.py

```diff
@@ -113,17 +132,36 @@ def dispatch(self, request, *args, **kwargs):
 
         return response
 
-    def _decompose_access_token_response(self, request, response):
+    def _decompose_access_token_response(self, adapter, response):
         """ Decomposes the access token in the request to an expiration date, scopes, and User. """
         content = json.loads(response.content)
         access_token = content['access_token']
         scope = content['scope']
-        access_token_obj = self.get_adapter(request).get_access_token(access_token)
-        user = access_token_obj.user
         scopes = scope.split(' ')
```

Comment:
Question: So I noticed that scopes are separated by commas in the Django admin, but then separated by space in the tokens.  Does the code parse the scopes appropriately at each point?  Also, does the `ApplicationAccess` model strip any whitespace that the admin may have put in between the commas?

Reply:
Yes, the `ListCharField` I'm using on the `ApplicationAccess` model handles parsing and validation of the comma-separated list entered in Django admin.

Reply:
ah great.